src/components/Project/ProjectHeader/index.tsx

```diff
@@ -37,8 +37,8 @@ export default function ProjectHeader({
   } = useContext(ThemeContext)
 
   const isMobile = useMobile()
-  const { data: ownerIsGnosisSafe, isLoading: ownerIsGnosisSafeLoading } =
-    useAddressIsGnosisSafe(projectOwnerAddress)
+  const { data: gnosisSafe, isLoading: gnosisSafeLoading } =
```

Comment:
Required signature threshold only available from `safe` object, not on individual transactions. So now just loading the safe and checking if exists rather than just a `boolean` in the case of `useAddressIsGnosisSafe`

Reply:
got it